backup_and_restore/application_backup_and_restore/installing/oadp-12-data-mover-ceph-doc.adoc

```diff
@@ -0,0 +1,62 @@
+[id="oadp-12-data-mover-ceph-doc"]
+= Using OADP 1.2 Data Mover with Ceph storage
+include::_attributes/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+You can use OADP 1.2 Data Mover to backup and restore application data for clusters that use CephFS, CephRBD, or both.
+
+OADP 1.2 Data Mover leverages Ceph features that support large-scale environments. One of these is the shallow copy method, which is available for {product-title} 4.12 and later. This feature supports backing up and restoring `StorageClass` and `AccessMode` resources other than what is found on the source persistent volume claim (PVC).
+
+[IMPORTANT]
+====
+The CephFS shallow copy feature is a back up feature. It is not part of restore operations.
+====
+
+include::modules/oadp-ceph-prerequisites.adoc[leveloffset=+1]
+
+[id="defining-crs-for-12-data-mover"]
+== Defining custom resources for use with OADP 1.2 Data Mover
+
+When you install {rh-storage-first}, it automatically creates default CephFS and a CephRBD `StorageClass` and `VolumeSnapshotClass` custom resources (CRs). You must define these CRs for use with OpenShift API for Data Protection (OADP) 1.2 Data Mover.
+
+After you define the CRs, you must make several other changes to your environment before you can perform your back up and restore operations.
+
+include::modules/oadp-ceph-preparing-cephfs-crs.adoc[leveloffset=+2]
+include::modules/oadp-ceph-preparing-cephrbd-crs.adoc[leveloffset=+2]
```

Comment:
VSClass resource defined in this file doesn't have default annotation

Reply:
annotations:
    snapshot.storage.kubernetes.io/is-default-class: true

Reply:
true -> "true"